### PR TITLE
PIP-1583-pbam

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: wdl/tasks/cat.wdl [hunk 1]
```diff
@@ -0,0 +1,28 @@
+version 1.0
+
+
+task cat {
+    input {
+        Array[File] files
+        String out = "concatenated"
```
Comment (paul-sud) [inline wdl/tasks/cat.wdl]: I think this name could be more descriptive:
```suggestion
        String output_filename = "concatenated"
```

File: docs/reference.md [hunk 2]
```diff
@@ -157,6 +158,20 @@ Kallisto quantifier makes use of average fragment lenghts and standard deviation
...
+
+#### Enabling the pBAM option
+To enable pBAM producing the following additional inputs are required:
+
+* `rna.produce_pbams` needs to be set to `true`.
+* `rna.reference_genome` The reference genome fasta compressed with gzip. The human reference genome used by ENCODE can be found [here](https://www.encodeproject.org/files/GRCh38_no_alt_analysis_set_GCA_000001405.15/).
+* `rna.reference_transcriptome` The reference transcriptome fasta compressed with gzip. The version should match the genome annotation version. The human version 29 transcriptome corresponding to ENCODE4 data can be found LINK HERE.
```
Comment (paul-sud) [inline docs/reference.md]: Still need the link?

**Review by paul-sud — CHANGES_REQUESTED** [review]
A couple small things, mostly looks good though.

File: docs/reference.md (hunk 2 above)
Comment (ottojolanki) [reply docs/reference.md]: thanks for catching this, it was not uploaded yet when I made the PR and forgot about it during last week..

Comment (ottojolanki) [reply docs/reference.md]: Added the link. Meenakshi will release the transcriptome as soon as checkfiles is done with it.

**Review by paul-sud — APPROVED** [review]

**Pull request merged**